src/content/docs/guides/development-workflow.mdx

```diff
@@ -0,0 +1,145 @@
+1. Work on task
+1. Open a pull request to merge changes into trunk
+1. Squash and merge after ci checks & code review
+
+Note that with the described workflow, the trunk (e.g. main) is always in a
+releasable state. This puts pressure on continuous integration (CI) checks to ensure
+that the code on the trunk is always in a healthy, deployable state. Checks that would
+typically run as part of the continuous integration process include:
+
+- Linting
+- Formatting
+- Unit tests
+- Integration tests
+- Test coverage reports
+
+You can refer to the following GitHub Actions workflow for an example.
+
+<CICheckSnippet />
+
+## Release Phase
+
+When the team is ready to distribute a new version of the app, a release is
+created. In some cases, releases are created on a regular cadence (e.g. weekly,
+bi-weekly, monthly) while in other cases, releases are created on an ad-hoc
+basis.
+
+The release workflow can be broken down into the following steps:
+
+1. Create a release branch (e.g release/v1.0.0) from trunk (main)
+1. Tag the commit (e.g v1.0.0)
+1. The release workflow is triggered and generates signed artifacts
+1. Distribute the artifacts (e.g. via AppStore, PlayStore, etc.)
+
+You can refer to the following GitHub Actions workflow for an example.
+
+<RemoteCode
+  url="https://raw.githubusercontent.com/felangel/flutter_and_friends/refs/heads/main/.github/workflows/release.yaml"
+  title=".github/workflows/release.yaml"
+/>
+
+When the release workflow has finished running, the artifacts are ready to be
+distributed to customers via the appropriate channels (e.g. AppStore, PlayStore,
+etc.).
+
+![Release Workflow](https://github.com/user-attachments/assets/7c9cf689-bd07-40c0-ad0b-e08935cb4424)
+
+:::note
+For more information about configuring signing refer to the following links:
+
+- [Create an Upload Keystore (Android)](https://docs.flutter.dev/deployment/android#create-an-upload-keystore)
+- [Installing an Apple Certificate on MacOS Runners (iOS)](https://docs.github.com/en/actions/use-cases-and-examples/deploying/installing-an-apple-certificate-on-macos-runners-for-xcode-development)
+
+:::
+
+## Patch Phase
+
+Sometimes, despite lots testing, critical bugs can creep into the app. Shorebird
+allows you to fix these bugs and distribute the patches to customers devices
+immediately rather having to wait for releases to be approved and distributed.
+
+The patch workflow can be broken down into the follow steps:
+
+1. Apply the hotfix to the trunk (identical to the development workflow above)
+   1. Create a branch from trunk (main)
+   1. Fix the bug
+   1. Open a pull request to merge changes into trunk
+   1. Squash and merge after ci checks & code review
+1. Cherry pick the commit into the desired release branch(es)
+1. Tag the commit using -hotfix.x (e.g v1.0.0-hotfix.1)
+1. The patch workflow is triggered and uploads the hotfix to the staging environment
+1. Preview the release in the staging environment & validate the fix
```

Comment:
Preview the release in the staging environment how?

Reply:
Generally, there's a lot happening in the yaml below. We should definitely include that (people love examples they can copy and tweak), but it'd be good call out specific commands for these steps, either in this list or as comments in the yaml